opal/datatype/opal_datatype_pack.h

```diff
@@ -27,6 +27,8 @@
     CONVERTOR->cbmemcpy( (DST), (SRC), (BLENGTH), (CONVERTOR) )
 #endif
 
+#define opal_datatype_pack_threshold 16 // for compile, use different value later
```

Comment:
why is this here ?

Reply:
Because you didn't define this in any file. It will cause compile error.

Reply:
hmm, did I missed my git add. let me check.